targets/TARGET_STM/can_api.c

```diff
@@ -778,6 +778,7 @@ void can_irq_init(can_t *obj, can_irq_handler handler, uint32_t id)
 {
     irq_handler = handler;
     can_irq_ids[obj->index] = id;
+    obj->rxIrqStatus = false;
```

Comment:
DISABLED?

Reply:
I think I suggested using `true` and `false` in a previous review, but that was supposed to be in conjunction with changing the variable name to `rxIrqEnabled`/`rxIrqDisabled`.

Reply:
Updated the code as per the comment.